quetz/tasks/mirror.py

```diff
@@ -292,30 +329,17 @@ def initial_sync_mirror(
             else:
                 logger.debug(f"updating package {package_name} form {arch}")
 
-            try:
-                remote_package = remote_repository.open(path)
-            except RemoteServerError:
-                logger.error(f"remote server error when getting a file {path}")
-                continue
+            update_batch.append(path)
+            update_size += metadata['size']
 
-            files = [remote_package]
-            try:
-                handle_package_files(
-                    channel_name,
-                    files,
-                    dao,
-                    auth,
-                    force,
-                )
-                any_updated = True
-            except Exception as exc:
-                logger.error(
-                    f"could not process package {package_name} from channel"
-                    f"{channel_name} due to error {exc} of "
-                    f"type {exc.__class__.__name__}"
-                )
-                if not skip_errors:
-                    raise exc
+            if len(update_batch) >= max_batch_length or update_size >= max_batch_size:
+                logger.debug(f"Executing batch with {update_size}")
+                any_updated = handle_batch(update_batch)
+                update_batch = []
+                update_size = 0
+
+        # handle final batch
+        any_updated = handle_batch(update_batch)
```

Comment:
probably you want to do:
```suggestion
        any_updated = any_updated or handle_batch(update_batch)
```

Reply:
I am using bitwise operator now (`|=`)